README.md

```diff
@@ -10,7 +10,16 @@ computer languages and build transformers, interpreters, compilers and other
 tools easily.
 
 Peggy is the successor of [PEG.js](https://github.com/pegjs/pegjs) which had been abandoned by its maintainer.
-Peggy version 1.x.x is backwards compatible with the most recent PEG.js release. Switching from PEG.js to Peggy should be as simple as uninstalling `pegjs` and installing `peggy` instead.
+
+## Migrating from PEG.js
+
+Peggy version 1.x.x is API compatible with the most recent PEG.js release.
+Follow these steps to upgrade:
+
+1. Uninstall `pegjs` (and `@types/pegjs` if you're using the DefinitelyTyped type definitions - we now include type definitions as part of peggy itself).
```

Comment:
Do we need to add a `types` field to package.json?

Reply:
No, TypeScript should detect the definitions automatically. It would be a good idea to do an `rc` release first just to make sure everything is working correctly though.

Reply:
Feel free to close #45 without change if everything works.  See email for next steps.

Reply:
I've added it just in case, I've probably been using the automatically-supported pattern in the past without knowing it. See #46